### ISSUE TITLE
Picture missing

### ISSUE THREAD

**Issue by superbonaci** [akamhy/waybackpy#191]
In the page https://pypi.org/project/waybackpy/ the image is not displayed.

https://pypi-camo.freetls.fastly.net/1de38d1188acafc8527a81ed7e31397c6763b4c0/68747470733a2f2f6769746875622e636f6d2f616b616d68792f7761796261636b70792f776f726b666c6f77732f54657374732f62616467652e737667

**Comment by ExplodingCabbage** [comment]
Specifically: the "unit tests" image is missing. It's broken in the README when viewed on GitHub, too - we just get the alt text. On GitHub, the img's src points to an svg in the repo that doesn't exist.

<img width="1712" height="1693" alt="Image" src="https://github.com/user-attachments/assets/792c288d-3c36-4ace-bd2a-0b347908b479" />
